pages/index.tsx

```diff
@@ -16,23 +15,25 @@ export default function Home() {
             <title>Create Next App</title>
             <link rel="icon" href="/favicon.ico" />
           </Head>
-
           <main className={styles.main}>
-            <Button primary label="Primary" />
+            {/* <Button primary label="Primary" />
             <Button primary={false} label="Secondary" />
-            <OpenNewPage />
-            <div>
-              <a href="/mainCard">
-                press here to navigate to the Main Card Page (temporary link)
-              </a>
-            </div>
+            <OpenNewPage /> */}
           </main>
           <div>
             <Greetings />
           </div>
-          <div>
-            <Form />
+          <div className={styles.btn}>
+            <a href="/Form">
+              <Btn label="Today I am out of ..." />
+            </a>
+            <a href="/mainCard">
+              <Btn label="Discover New Ingredients!" />
+            </a>
```

Comment:
These should be `Link` elements. Take a look here:
https://nextjs.org/docs/api-reference/next/link